README.md

```diff
@@ -17,9 +17,11 @@ yarn install
 Start local dev server for Vue client, with Hot Module Replacement.
 ```
 # Start client
-yarn workspace client run dev
-
+yarn workspace client dev
```

Comment:
```suggestion
yarn workspace client run dev
```

Need that `run` in there.

Reply:
Oh you're totally right, I didn't realize that. Feel free to take it out again if you prefer.